projects/Mallard/src/helpers/push-tracking.ts

```diff
@@ -14,10 +38,32 @@ interface Tracking {
 const getPushTracking = async (): Promise<string | null> =>
     AsyncStorage.getItem(PUSH_TRACKING_KEY)
 
+// Only get the start and end of the push notification process
+const getDiagnosticPushTracking = async () => {
+    try {
+        const pushTrackingString = await AsyncStorage.getItem(PUSH_TRACKING_KEY)
+        if (!pushTrackingString) {
+            return null
+        }
+
+        const pushTracking = JSON.parse(pushTrackingString)
+        return pushTracking.find(
```

Comment:
My understanding might be wrong, if we want the first and last `state` of the push download process then we probably need the first and last item from the `pushTracking` array, no?

Reply:
Nope - because its one log for a 7 day period. Its a daily process.